components/CovidMap.js

```diff
@@ -319,21 +316,23 @@ class CovidMap extends React.Component {
       thirdLegendText: '',
       infowindow: null,
     };
+    this.updateData = this.updateData.bind(this);
   }
 
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
I think this is fine as a solution. You might check out NextJS [getIntialProps](https://nextjs.org/docs/api-reference/data-fetching/getInitialProps) function.It may be a better way to handle this in the future, I'm not sure.

Either way I don't think that needs to stop this fix from going through.